src/js/core.js

```diff
@@ -1088,6 +1138,66 @@
                 target.innerHTML = html;
                 this.events.updateInput(target, { target: target, currentTarget: target });
             }
+        },
+
+        addElements: function (elements) {
+            var filtered = [];
+
+            // We want always an array with our input
+            if (!elements.length) {
+                elements = [elements];
+            }
+
+            // Filter the input, we want to include every element only once!
+            elements.forEach(function (element) {
+                if (element.getAttribute('data-medium-editor-element')) {
+                    return;
+                }
+
+                filtered.push(element);
+            });
+
+            // Do we have elements to add now?
+            if (filtered.length === 0) {
+                return false;
+            }
+
+            // Add new elements to our internal elements array
+            this.elements = this.elements.concat(filtered);
+
+            // Initialize all new elements (we check that in those functions don't worry)
+            initElements.call(this);
+            reAttachHandlers.call(this);
+        },
+
+        cleanupElements: function () {
```

Comment:
What would you think about reducing this method to just be something like `removeElements` where you pass the elements you want to remove into the method directly?  Seems like something like finding if the element exists in the DOM or not, or the `checkExistenceTagName` option might be better to leave outside of the editor?